runtime/compiler/control/JITServerHelpers.hpp

```diff
@@ -146,6 +146,14 @@ class JITServerHelpers
    // Helper routine to generate a unique ID for the client or server
    static uint64_t generateUID();
 
+   static size_t getFullClassNameLength(const J9ROMClass *romClass, const J9ROMClass *baseComponent,
```

Comment:
If we're assuming that the full class name's length fits in a `uint32_t`, this function could return that type instead.

Reply:
Updated in latest force push.